roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -41,6 +43,8 @@
     gpgcheck: no
     enabled: yes
   when:
+    - katello_repositories_version != "nightly"
+    - katello_repositories_version is version('4.0', '<=')
     - katello_repositories_pulp_version == "nightly"
```

Comment:
I don't think we use pulp nightly anywhere now. Should we drop it?

Reply:
yeah, good idea

Reply:
actually, pipelines/vars/katello_devel.yml still refers to it. I'll follow up with katello about this.